### PR TITLE
Predict module

### PR DESCRIPTION
**Pull request by miltminz**
This PR introduces important new functionalities like:
- `queries_api.py`: Query FWI and ERA5 data from CDS API
- `predict.py`: introduces predict object `PyroRisk` which instantiates with a trained model and returns predictions for an input day
- functions that takes care of the ingestion and transform the data in order to be predicted by the model
- unitary tests

### CODE REVIEW

File: pyro_risks/datasets/queries_api.py [hunk 1]
```diff
@@ -0,0 +1,435 @@
...
+        },
+        os.path.join(output_path, f"era5t_{year}_{month}_{day}.nc"),
+    )
+    # TODO : take only needed variables for the model
+
+
+def call_fwi(output_path, year, month, day):
+    """Get data from Fire danger indices historical data from the Copernicus Climate Data Store.
+
+    Information on FWI can be found here:
```
Comment (frgfm) [inline pyro_risks/datasets/queries_api.py:400]: Trailing whitespace here apparently

**Review by frgfm — CHANGES_REQUESTED** [review]
Thanks for the PR! Only a little trailing whitespace, but most importantly: adding cdsapi to requirements :pray:

The rest looks good to me!

File: pyro_risks/datasets/queries_api.py (hunk 1 above)
Comment (miltminz) [reply pyro_risks/datasets/queries_api.py]: yes thanks, it's fixed

**Comment by chloeskt** [comment]
Thank you for this PR @miltminz, you did an amazing job ! 🤗  All good for me 👍

**Review by chloeskt — APPROVED** [review]

**Review by frgfm — APPROVED** [review]
Thank you for the PR! All good on my side, we only have to keep in mind that we'll need to optimize the tests that are consuming precious free Github workflows minutes :sweat_smile: 

I'll open a PR!

**Pull request merged**